<launch>
  <param name="robot_description" 
         command="$(find xacro)/xacro.py '$(find aerial_robot_model)/robots/test/test.urdf.xacro'" />
  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />


</launch>
